pkg/controller/vault.go

```diff
@@ -48,9 +48,11 @@ import (
 const (
 	EnvVaultAddr            = "VAULT_API_ADDR"
 	EnvVaultClusterAddr     = "VAULT_CLUSTER_ADDR"
+	EnvVaultCACert          = "VAULT_CACERT"
 	VaultClientPort         = 8200
 	VaultClusterPort        = 8201
 	vaultTLSAssetVolumeName = "vault-tls-secret"
+	TLSCACertKey            = "cacert.crt"
```

Comment:
If you use `ca.crt` instead of `cacert.crt`, it'd match the convention that [cert-manager](https://cert-manager.io) has set up with its TLS secrets. I'm planning on submitting a patch to allow pulling the CA bundle out of the secret so that I can use `cert-manager` to bootstrap the server TLS. It would be convenient if the secret generated here followed the same convention.